benchmark/cmd/stream.go

```
@@ -0,0 +1,296 @@
+		Term:   "xterm-256color",
+		Env:    map[string]string{"BENCH": "true"},
+	}
+	
+	session, err := c.CreateSession(config)
+	if err != nil {
+		result.Errors = append(result.Errors, fmt.Errorf("create session: %w", err))
+		return result
+	}
+	
+	result.SessionID = session.ID
+	defer c.DeleteSession(session.ID)
```

golangci-lint (1.64.8)

Error return value of `c.DeleteSession` is not checked

(errcheck)